content/soarian/dstu2/care-provision/careplan.md

```diff
@@ -0,0 +1,96 @@
+
+
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+
+## Search
+	
+	GET /CarePlan?:parameters
+	
+_Implementation Notes_
+
+* The API returns the most current careplan list.
+
+* The Narrative includes additional information pertaining to the careplan, and should be shown to the user to ensure completeness of clinical content and context.
+
+### Parameters
+
+|Name |Required? | Type | Description
+| --- | --- | --- | --- |
+| patientId | Y | [reference](http://hl7.org/fhir/DSTU2/search.html#reference) | The patient identifier provided in a pre-requisite authorization step. |
+| date | N | [date](http://hl7.org/fhir/DSTU2/search.html#date) as adjusted per implementation notes | Null or specific date or a date range. |
+
+Notes:
+
+* See details regarding special [date](../../#special-information-regarding-date-parameters) requirements.
+* If the query date parameter is null in the query, the API will return qualifying records with dates less than or equal to the date of query as limited by the lesser of 1000 records or 30 days inclusive of the query date.  
+* If a single date parameter is included, the response will include qualifying records dated that day as limited by 1000 records.  
+* If a valid date range is used in the query, the API will return qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit date specified.
+* If a date range is used greater than 30 days, the response will include qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit date specified and include in the informational response in the narrative extension that the date range entered is greater than 30 days and please refine your response.
+* If an invalid date range is specified in the query, the API will error.
+
+
+### Headers
+
+<%= headers %>
+ 
+### Example 
+    
+#### Request
+
+	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc/CarePlan?patientId=D4292B7B-AEDF-4CEF-B783-BD3AA1B0DD27
+
+
+#### Response  
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_CARE_PLAN_ENTRY) %> 
+
+ 
+### Errors and Informational messages
+The common [errors and informational messages](../../common-errors) can be returned.
+
+These additional informational messages may be returned within the [bundle extension]:
+
+| Code | Message |
+| --- | --- |
+| papi027 | No relevant clinical data exists.|
+| papi026 | The information provided is the latest configured data available at the time of charting and may not be reflective of the current clinical state of the care plan. It is the responsibility of the patient or patient representative to confirm care plan status with appropriate care providers.|
```

Comment:
Could you please specify what exactly the change needed here ? The content seems to be same as original.